src/Test.UnitTests.Sarif.Driver/TestRule.cs

```diff
@@ -0,0 +1,241 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Composition;
+using System.IO;
+using System.Resources;
+using FluentAssertions;
+using Microsoft.CodeAnalysis.Sarif.Driver;
+using Microsoft.CodeAnalysis.Test.UnitTests.Sarif.Driver.Sdk;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    [Export(typeof(ReportingDescriptor)), Export(typeof(IOptionsProvider)), Export(typeof(Skimmer<TestAnalysisContext>))]
+    internal class TestRule : TestRuleBase, IOptionsProvider
```

Comment:
>TestRule [](start = 19, length = 8)

SimpleTestRule (no longer simple) has been renamed TestRule (with the exception raising logic merged into it).